packages/eslint-plugin-calcite-components/src/rules/enforce-ref-last-prop.ts

```diff
@@ -17,20 +17,46 @@ const rule: Rule.RuleModule = {
       JSXIdentifier(node) {
         const openingElement = node.parent as JSXOpeningElement;
         if (openingElement.type === "JSXOpeningElement") {
-          const attributes: string[] = [];
+          const attributes: (JSXAttribute | JSXSpreadAttribute)[] = [];
```

Comment:
Can you use `map` to build this vs creating the array and populating separately? Something like this?

```tsx
const attributes: (JSXAttribute | JSXSpreadAttribute)[] = openingElement.attributes.map((attr: JSXAttribute | JSXSpreadAttribute) => {
  if (attr.type === "JSXAttribute" && attr.name?.type === "JSXIdentifier") {
    return attr;
  }
}).filter(Boolean);
```